web/forms.py

```
@@ -946,6 +946,14 @@ class TeachForm(forms.Form):
         )
     )
 
+    def __init__(self, *args, **kwargs):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)